template-parts/header.php

```diff
@@ -26,35 +36,42 @@
                         foreach ( $tags as $tag ) :
                         $tag_link = get_tag_link( $tag->term_id );
                     ?>
-                    <li class="m-2">
+                    <li>
                         <a href='<?php echo $tag_link; ?>' title='<?php echo $tag->name; ?>'>#<?php echo $tag->name ?></a>
                     </li>
                     <?php
                         endforeach;
                     ?>
                     </ul>
                 </nav>
-                <nav class="top-nav">
+
+                <nav class="text-sm flex gap-5 items-center">
                     <?php
                         wp_nav_menu( array(
                             'theme_location' => 'top-menu',
                             'fallback_cb' => false,
-                            'menu_class' => 'list-none flex space-x-4',
+                            'menu_class' => 'list-none flex gap-4',
                             'items_wrap' => '<ul id="%1$s" class="%2$s">%3$s</ul>',
                         ) );
                     ?>
+
+                    <button>
+                        <span></span>
+                        <span>Подкрепи ни</span>
+                    </button>
                 </nav>
-            </div> 
+            </div>
         </div>
 
         <!-- Main Navigation -->
         <div class="main-nav">
-            <div class="flex">
-                <a class="mr-4" href="<?php echo get_home_url(); ?>">
-                    <h2>CarLife by Dani Logo</h2>
+            <div class="flex items-center gap-6">
+                <a class="w-full h-full max-w-[224px]" href="<?php echo get_home_url(); ?>">
+                    <img src="http://carlifebydani.local/wp-content/uploads/2023/12/logo-1.png" alt="Logo">
```

Comment:
Static images like loge etc should be part of the content
create /images folder in the theme root and load from there
we should you svg whenever possible 
```<img src="<?php echo get_stylesheet_directory_uri(); ?>/images/logo.svg" />```